pkgs/by-name/ru/runapp/package.nix

```
@@ -0,0 +1,50 @@
+{
+  fetchFromGitHub,
+  gcc15,
+  lib,
+  nix-update-script,
+  overrideCC,
+  pkg-config,
+  stdenv,
+  systemd,
+}:
+let
+  stdenv' = overrideCC stdenv gcc15;
+in
+stdenv'.mkDerivation rec {
+  pname = "runapp";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "c4rlo";
+    repo = pname;
```

- NixOS/nixpkgs/issues/277994
```suggestion
    repo = "runapp";
```